frequently-asked-questions.md

```diff
@@ -79,9 +79,9 @@ Yes. CockroachDB distributes transactions across your cluster, whether it’s a
 Yes. Every transaction in CockroachDB guarantees ACID semantics.
 
 - **Atomicity:** Transactions in CockroachDB are “all or nothing.” If any part of a transaction fails, the entire transaction is aborted, and the database is left unchanged. If a transaction succeeds, all mutations are applied together with virtual simultaneity.   
-- **Consistency:** In CockroachDB, non-distributed transactions use the [Raft consensus algorithm](https://raft.github.io/) to ensure the most recent data is read. Distributed transactions achieve consistency by relying on a limited degree of clock synchronization in conjunction with transaction restarts in the event of contention. 
-- **Isolation:** By default, transactions in CockroachDB use serializable snapshot isolation (SSI). This means that concurrent transactions on the same data, whether reads or writes, will never result in anomalies. We also provide snapshot isolation (SI), which is more performant with high-contention workloads, although it exhibits rare classes of anomalies not present in SSI.
-- **Durability:** In CockroachDB, once a transaction has been committed, the results are stored on-disk permanently. Replication provides availability and the [Raft consensus algorithm](https://raft.github.io/) guarantees consistency between the replicas.
+- **Consistency:** SQL operations never see any intermediate states and move the database from one valid state to another, keeping indexes up to date. Operations will always see the results of previously completed statements on overlapping keys. This is achieved using a limited degree of clock synchronization across the cluster.
```

Comment:
-I'd remove the last line because the implementation itself is quite complex and one line doesn't give it justice.
-change overlapping keys to overlapping data.
- The databases ensures that all SQL specified constraints such as unique columns are maintained